docs/recipes/integration-test/example-lh-auth.test.js

```diff
@@ -84,6 +85,7 @@ describe('my site', () => {
       args: [`--remote-debugging-port=${CHROME_DEBUG_PORT}`],
       headless: !process.env.DEBUG,
       slowMo: process.env.DEBUG ? 50 : undefined,
+      executablePath: getChromePath(),
```

Comment:
This is unfortunate to have in our example code. Users likely want to use the chrome provided by puppeteer for their tests.

Reply:
Yes but it's in the example `test.js` file so I'm not too worried about it. There is already a lot test boilerplate in this file.